docs/content/preview/yugabyte-voyager/migrate/assess-migration.md

```diff
@@ -91,7 +159,7 @@ A sample Migration Assessment report is as follows:
         --source-db-schema schema1,schema2 --export-dir /path/to/export/dir
         ```
 
-    1. **Without source database connectivity**: In situations where direct access to the source database is restricted, there is an alternative approach. Voyager includes packages with scripts present at `/etc/yb-voyager/gather-assessment-metadata/postgresql`. You can perform the following steps with these scripts.
+    1. **Without source database connectivity**: In situations where direct access to the source database is restricted, there is an alternative approach. Voyager includes packages with scripts for both PostgreSQL and Oracle present at `/etc/yb-voyager/gather-assessment-metadata`. You can perform the following steps for PostgreSQL with these scripts (similar for Oracle).
 
         1. Copy the scripts to a machine which has access to the source database.
```

Comment:
Can you change this step to this:

On a machine which has access to the source database, copy the scripts, and install dependencies - psql, pg_dump >=14. Alternatively, you can install yb-voyager on the machine to automatically get the dependencies installed.